### PR TITLE
[Tensorflow][Release]Releasing py36 and py37 images

### PR DESCRIPTION
**Pull request by TusharKanekiDey**
*Issue #, if available:*

## Checklist
- [ ] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [build] | [test] | [build, test] | [ec2, ecs, eks, sagemaker]

*Description:*
We are releasing currently only py36 and py37 training Images currently.

*Tests run:*

*DLC image/dockerfile:*

*Additional context:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: release_images.yml [hunk 1]
```diff
@@ -2,9 +2,25 @@
...
+    version: "1.15.3"
+    training:
       device_types: ["cpu", "gpu"]
-      python_versions: ["py37"]
+      python_versions: ["py37", "py36"]
       os_version: "ubuntu18.04"
-      cuda_version: "cu102"
+      cuda_version: "cu100"
+      example: False        # [Default: False] Set to True to denote that this image is an Example image
+      disable_sm_tag: True  # [Default: False] Set to True to prevent SageMaker Abbreviated Tags from being attached
```
Comment (arjkesh) [inline release_images.yml:12]: just double checking, we are not releasing a sm tag for either py37 or py36?

Comment (YYStreet) [inline release_images.yml:12]: Why are we disabling sm_tag? Checked pro repo for tf1.15.2 and it is enabled

Comment (TusharKanekiDey) [reply release_images.yml]: We are releasing SM tags, I have made the required changes

Comment (TusharKanekiDey) [reply release_images.yml]: Sorry, I have updated it.

**Comment by YYStreet** [comment]
> Are py27 images not a part of the release?

We are missing gpu py2. We are considering to include traing py2 cpu/gpu together with inference images in the following release

**Comment by YYStreet** [comment]
Please wait for CDK change being into effect then merge this PR

**Pull request merged**